test/test_data/specifications/vesuvio_specification.json

```diff
@@ -1,3 +1,5 @@
 {
-  "enabled": false
+  "enabled": true,
```

Comment:
I thought the specifications were now loaded and saved into the database. If so shouldn't all these json files be deleted instead? Otherwise it looks good.

Reply:
Not for testing purposes no, these are exclusively for tests.